### ISSUE TITLE
DetailsList: Screenreader announces the entire row's contents in one go

### ISSUE THREAD

**Issue by nkalpana** [OfficeDev/office-ui-fabric-react#11486]
Screenreader announces the entire row's contents in one go

### Environment Information

- **Package version(s)**: Latest version used by code samples exported to codepen from Controls playground
- **Browser and OS versions**: Windows 10

### Describe the issue:

When a user scans a row, using Screenreader and Narrator on a DetailsList table, it reads all the columns contents in one go, without a pause, and then it says "row" in the end.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/nkalpana/pen/BayQmQB

#### Actual behavior:

Steps to repro:
a) Open codepen from above
b) Open Narrator and OnScreen keyboard. Scan a row
c) Observe that it says Excel Icon/Word Icon, file name, created time, and other columns in one go and then softly finishes with "row" at the end. This can be confusing to user.

When a user scans a row, using Screenreader and Narrator on a DetailsList table, it reads all the columns contents in one go, without a pause, and then it says "row" in the end.

#### Expected behavior:

The announcement should be in such a way a user can understand. Announce which column, and columns' content, and with enough pauses.

**Comment by micahgodbolt** [comment]
This is the expected default behavior of the control, and relied upon by a large number of apps, so we don't plan on changing the default behavior.

If the contents of your details row is bit verbose, you can modify the aria label string via the `getRowAriaLabel` prop

```
getRowAriaLabel={(item) => item.name}
```
